src/frontCANnode/src/main.c

```diff
@@ -0,0 +1,98 @@
+#include "serial.h"
+#include "types.h"
+#include "input.h"
+#include "output.h"
+#include "adc.h"
+
+#include "timer.h"
+
+#include "sysinit.h"
+#include "chip.h"
+
+#include "CANlib.h"
+
+#define SERIAL_BAUDRATE 57600
+#define CAN_BAUDRATE 500000
+
+const uint32_t OscRateIn = 12000000;
+volatile uint32_t msTicks;
```

Comment:
As I was saying, this can be redeclared in a header file called tick.h or something and included here and wherever else needed.

So keep line 18 as is and add a header file with an extern volatile uint32_t msTicks; that you will include in main,c and input.c and wherever else. That allows them to access the same msTicks.